deployments/strongloop.sh

```diff
@@ -7,16 +7,16 @@
 # * SL_DEPLOY_URL
 #
 # Include in your builds via
-SL_BUILD_VERSION=${SL_NPM_BUILD_VER:?'latest'}
-SL_DEPLOY_VERSION=${SL_NPM_DEPLOY_VER:?'latest'}
+SL_BUILD_VERSION=${SL_BUILD_VERSION:?'latest'}
+SL_DEPLOY_VERSION=${SL_DEPLOY_VERSION:?'latest'}
```

Comment:
Yes, these can be set by the user to require a specific version. If they aren't set we default to the `latest` version.